python/az/aro/azext_aro/custom.py

```diff
@@ -145,6 +147,7 @@ def aro_create(cmd,  # pylint: disable=too-many-locals
             pod_cidr=pod_cidr or '10.128.0.0/14',
             service_cidr=service_cidr or '172.30.0.0/16',
             outbound_type=outbound_type or '',
+            preconfigured_nsg='Enabled' if enable_preconfigured_nsg else 'Disabled',
```

Comment:
this is good to merge since it follows suit, but I created a follow-up item for someone on our team to tackle: https://issues.redhat.com/browse/ARO-4250